actors/init/src/v0/state.rs

```diff
@@ -2,8 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use cid::Cid;
+use fil_actors_shared::v9::{make_map_with_root_and_bitwidth, shared::HAMT_BIT_WIDTH};
```

Comment:
Are you sure the v9 versions of these functions are identical to the v0 versions?